app/views/pages/_contributor_blurb.html.erb

```diff
@@ -18,7 +18,7 @@
         <div class="contributorModalBodyCopy">
           <div class="contributorModalHeader">
             <div class="contributorModalBodyPhoto showOnSmall">
-                <img class="contributorAvatar" src="#{ProfilePicture.normalize_url(blurb["image"])}">
+                <img class="contributorAvatar" src="#{ProfilePicture.normalize_url(blurb["image"])}" alt="contributorAvatar">
```

Comment:
So it looks like we already pass in alt text for this avatar! We do so earlier in the file, [here](https://github.com/ifmeorg/ifme/pull/1608/files#diff-b509bbd2f7a3498d3d09753bf546fa9dR7)!

Reply:
all fixed.